src/linter/ui5Types/fix/collections/sapUiCoreFixes.ts

```diff
@@ -112,9 +114,334 @@ t.declareModule("sap/ui/core/Core", [
 				return `${moduleIdentifier}.setTheme(${arg1})`;
 			},
 		})),
+		// Note that alternative replacement Component.get is meanwhile deprecated, too
 		t.method("getComponent", accessExpressionFix({
 			moduleName: "sap/ui/core/Component",
 			propertyAccess: "getComponentById",
 		})),
+		// Individual arguments must be mapped to "options" object
+		// The new API has no sync loading option, replacement is only safe when the options contain async:true
+		t.method("loadLibrary", callExpressionGeneratorFix({
+			moduleName: "sap/ui/core/Lib",
+			propertyAccess: "load",
```

Comment:
This parameter is not used by the generator-fix. This is an issue in the typings that I just corrected in https://github.com/SAP/ui5-linter/pull/714

```suggestion
```